index.html

```diff
@@ -2078,6 +2150,22 @@ <h2>
 such as Update or Deactivate, MUST clearly state these limitations.
       </p>
 
+      <p>
+Determining the authority of a party to carry out the operations is method-specific.
+For example, a <a>DID method</a> MAY:
+      </p>
+      <ul>
+        <li>
+use the <a>verification methods</a> listed under <a>authentication</a> to decide whether an update/deactivate operation is allowed; or
+        </li>
+        <li>
+use other constructs in the <a>DID Document</a> to decide this; or
```

Comment:
Hmm, we may want to mention: For example, a verification method specified under `capabilityInvocation` could be used to verify the invocation of a capability to update the DID Document.

Reply:
Updated